peribolos/knative-sandbox.yaml

```diff
@@ -701,7 +708,11 @@ orgs:
       Operations Writers:
         description: Grants write access to operations-related repositories.
         privacy: closed
-        repos: {}
+        repos:
+          monitoring: write
+        members:
+        - dprotaso
+        - upodroid
```

Comment:
My $0.02 is that for a single repo, we don't need to create a group. This list should be actual "approvers" in the Operations WG.

I'd add `upodroid` to OWNERS in the resulting repo rather than adding them to (possibly multiple) experiments here in the future.

Reply:
Actually thinking a bit - should we aim to move one off users out of OWNER files and into GitHub teams just for the sake of making removal/cleanup easier?